### PR TITLE
Continuous audio

### PR DESCRIPTION
**Pull request by sfalkena**
Pull request to merge continuous audio files to Master branch

### CODE REVIEW

File: Matlab/Final/EPO4_audio_record.m [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+    % Show the raw data
+    %for jj = 1:nMicrop        
+    %    figure(jj);
+    %    plot(RXr(:,jj));
+    %    grid on;
+    %end
+end
+
+% Save all the data and parameters code,Fs,nMicrop,nLoop,Nrp
```
Comment (sdelfos) [inline Matlab/Final/EPO4_audio_record.m]: I think you can delete the last two lines. For the standard recording and saving you can just use the epo4_audio_measure file.

File: Matlab/Final/Audio_continuous.m [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Audio_Settings;                     %Set parameters and turn on audio beacon
+i=0;
+for i=1:6
+    RXXr = EPO4_audio_record('B5_audio', 15000,7500,2500,'ebeb9a61',48e3,5,1,3);   %Records and saves recording in audiodata_B5_1
+    [x_cor(i),y_cor(i)] = Testfile(RXXr)                                          %Deletes file to prevent multiple files with same name
```
Comment (sdelfos) [inline Matlab/Final/Audio_continuous.m]: If you initialize with `x_cor = zeros(1,6); ` you will lose the annoying warning matlab throws saying that the size changes on every iteration. Should take a little less time to execute as well.

[the remark this answers is not in the document]
Comment (sfalkena) [reply Matlab/Final/Audio_continuous.m]: nope

Comment (sfalkena) [reply Matlab/Final/Audio_continuous.m]: We can do that while integrating

**Pull request merged**